astropy/coordinates/angles/angle_parsetab.py

```diff
@@ -18,7 +18,7 @@
 _lr_method = 'LALR'
 
 _lr_signature = 'COLON DEGREE EASTWEST HOUR MINUTE NORTHSOUTH SECOND SIGN SIMPLE_UNIT UFLOAT UINT\n            angle : sign hms eastwest\n                  | sign dms dir\n                  | sign arcsecond dir\n                  | sign arcminute dir\n                  | sign simple dir\n            \n            sign : SIGN\n                 |\n            \n            eastwest : EASTWEST\n                     |\n            \n            dir : EASTWEST\n                | NORTHSOUTH\n                |\n            \n            ufloat : UFLOAT\n                   | UINT\n            \n            colon : UINT COLON ufloat\n                  | UINT COLON UINT COLON ufloat\n            \n            spaced : UINT ufloat\n                   | UINT UINT ufloat\n            \n            generic : colon\n                    | spaced\n                    | ufloat\n            \n            hms : UINT HOUR\n                | UINT HOUR ufloat\n                | UINT HOUR UINT MINUTE\n                | UINT HOUR UFLOAT MINUTE\n                | UINT HOUR UINT MINUTE ufloat\n                | UINT HOUR UINT MINUTE ufloat SECOND\n                | generic HOUR\n            \n            dms : UINT DEGREE\n                | UINT DEGREE ufloat\n                | UINT DEGREE UINT MINUTE\n                | UINT DEGREE UFLOAT MINUTE\n                | UINT DEGREE UINT MINUTE ufloat\n                | UINT DEGREE UINT MINUTE ufloat SECOND\n                | generic DEGREE\n            \n            simple : generic\n                   | generic SIMPLE_UNIT\n            \n            arcsecond : generic SECOND\n            \n            arcminute : generic MINUTE\n            '
-
+    
```

Comment:
Did `ply` really add this?

Reply:
I deleted the file and then installed using build_ext then ran pytest. Pre-commit took a look at the file. I didn't touch it personally.

Reply:
Isn't the file excluded from `pre-commit`? It certainly should be, but perhaps the change here means the regex doesn't match any more?

Reply:
end-of-file-fixer and ruff are both not allowed to touch parsetab files.

Reply:
So nothing should be causing that except ply

Reply:
But it is odd

Reply:
I don't like the tab here, but we shouldn't be manually editing this file at all.